app/views/groups/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<% content_for :header_title, "Groups" %>
+
+<div id="groups" class='flex flex-col gap-3 m-3'>
+  <% @groups.each do |group| %>
+    <%= render group %>
+  <% end %>
+</div>
+
+<div class='bg-green-600 p-3 fixed w-full bottom-0'>
+  <div class='border p-3 border-white text-center sticky'><%= link_to "New Group", new_group_path, class: "text-white" %></div>
```

Comment:
- [ ] Well done on implementing this button, However, according to the instructions, after creating a group, the user should be redirected to the home page. But when I create the group, I'm not redirected to the home page. Instead, I'm redirected to the groups  show page:
![image](https://github.com/limavfabio/easy_pay/assets/45460192/872ead01-172e-41c8-9c24-1a898a03f9a5)